Pear/Cells/CommunityUserTableViewCell.swift

```diff
@@ -98,18 +98,24 @@ class CommunityUserTableViewCell: UITableViewCell {
         }
     }
 
-    func configure(with user: CommunityUser) {
-        if let firstName = user.firstName, let lastName = user.lastName {
-            nameLabel.text = "\(firstName) \(lastName)"
-            if let major = user.major, let gradYear = user.graduationYear, let hometown = user.hometown, let pronouns = user.pronouns, let userInterests = user.interests {
-                informationLabel.text = "\(major) · \(gradYear) · \(hometown) · \(pronouns)"
-                interests = userInterests
-                interestsCollectionView.reloadData()
-            }
-            if let profilePictureURL = URL(string: user.profilePictureURL ?? "") {
-                profileImageView.kf.setImage(with: profilePictureURL)
-            }
+    func configure(with user: UserProfile) {
+        nameLabel.text = "\(user.firstName) \(user.lastName)"
+        if let profilePictureURL = URL(string: user.profilePicUrl) {
+            profileImageView.kf.setImage(with: profilePictureURL)
         }
+        interests = user.interests
+        interestsCollectionView.reloadData()
+//        if let firstName = user.firstName, let lastName = user.lastName {
```

Comment:
I understand that there is messy code so I just want to ask if there is any reason you're keeping this behind (backend bug, something left to implement on our side, etc.)

Reply:
I mostly left this behind because I think the model changed a bit for this user table. And I forget which fields returned might be null in the new backend. I figure we will just tackle this in the future network integration.